core/layout_api.php

```diff
@@ -91,18 +93,26 @@ function layout_page_header_begin( $p_page_title = '' ) {
 		echo "\t", '<link rel="shortcut icon" href="', helper_mantis_url( $t_favicon_image ), '" type="image/x-icon" />', "\n";
 	}
 
-	# Advertise the availability of the browser search plug-ins.
-	$t_title = htmlspecialchars( config_get( 'search_title' ) );
-	$t_searches = array( 'text', 'id' );
-	foreach( $t_searches as $t_type ) {
+	if( auth_anonymous_enabled() ) {
 		echo "\t",
-			'<link rel="search" type="application/opensearchdescription+xml" ',
-				'title="' . sprintf( lang_get( "opensearch_{$t_type}_description" ), $t_title ) . '" ',
-				'href="' . string_sanitize_url( 'browser_search_plugin.php?type=' . $t_type, true ) .
-				'"/>',
+			'<link rel="sitemap" type="application/xml" title="Sitemap" ',
+			'href="', $t_path, 'sitemap.php', '">',
 			"\n";
 	}
 
+	# Advertise the availability of the browser search plug-ins.
```

Comment:
Is this a different feature?

Reply:
Good point, I'm trying to stick small changes together again... 😔 Moved to #2126.